### PR TITLE
Small fixes for 2.0.5 release

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

File: lib/cluster/corosync.c [hunk 1]
```diff
@@ -688,6 +688,18 @@ corosync_cluster_name(void)
...
+ * \internal
+ * \brief Check whether Corosync has any configuration keys with given prefix
+ *
+ * \param[in] prefix  Configuration key prefix
+ *
+ * \return -1 on error, 0 if no such key present, 1 if any such key present
+ * \note This function always returns the result of the first time it was
+ *       called, even if later called with a different section name (which
+ *       obviously should never be done), and even if the corosync configuration
+ *       has since been reloaded.
```
Comment (kgaillot) [inline lib/cluster/corosync.c:701]: Longer term, this should be refactored to address the issues in the note, but I wanted to minimize the changes for 2.0.5

**Comment by kgaillot** [comment]
@clumens, can you review when you get a chance?

**Comment by kgaillot** [comment]
retest this please

File: lib/cluster/corosync.c (hunk 1 above)
Comment (clumens) [reply lib/cluster/corosync.c:701]: Maybe toss a FIXME on here, unless you've got it on some sort of list.

**Comment by clumens** [comment]
Looks good to me.

File: lib/cluster/corosync.c (hunk 1 above)
Comment (kgaillot) [reply lib/cluster/corosync.c:701]: I do hope to get to it reasonably soon, just didn't want to go overboard with changes in 2.0.5

**Pull request merged**